content/metadata.yaml

```diff
@@ -70,6 +70,42 @@ authors:
     coi:
       string: "None"
       lastapproved: !!str 2017-05-26
+  - github: jlevy44
+    name: Joshua J. Levy
+    orcid: 0000-0001-8050-1291
+    email: joshua.j.levy.gr@dartmouth.edu
+    affiliations:
+      - Program in Quantitative Biomedical Sciences, Geisel School of Medicine at Dartmouth, Lebanon, NH
+    funders: Burroughs Wellcome Fund Big Data in the Life Sciences training grant at Dartmouth
+    v1: ""
+    v2: "Drafted one or more subsections"
+    coi:
+      string: "None"
+      lastapproved: !!str 2019-07-22
+  - github: brockclarke
+    name: Brock C. Christensen
+    orcid: 0000-0003-3022-426X
+    email: brock.christensen@dartmouth.edu
+    affiliations:
+      - Department of Epidemiology, Geisel School of Medicine, Dartmouth College, Lebanon, NH
+    funders: NIH R01CA216265, Dartmouth College Neukom Institute for Computational Science CompX award
+    v1: ""
+    v2: "Drafted one or more subsections"
+    coi:
+      string: "None"
+      lastapproved: !!str 2019-08-01
+  - github: AlexanderTitus
+    name: Alexander J. Titus
+    orcid: 0000-0002-0145-9564
+    email: alexander.j.titus@gmail.com
```

Comment:
@jlevy44 can you please update my email to Alexander.J.Titus@Dartmouth.edu. 

After that, I approve the update, and confirm that I have COI is "none". 

Thanks all.

Reply:
Done!